docs/tutorials/t-10.md

```diff
@@ -0,0 +1,118 @@
+- Setup the accounts on [`Algosigner`](https://github.com/PureStake/algosigner), [`Wallet Connect`](https://walletconnect.com/) wallet and [`MyAlgo Wallet`](https://wallet.myalgo.com/). It could be MainNet, TestNet or private net.
+
+## Steps
+
+In this tutorial we will use Algo Builder Wallet Widget to connect to various web wallets. Then we will show how to instrument the wallet to sign a multisignature transaction. See [Wallet Widget tutorial](https://developer.algorand.org/solutions/vuejs-multi-wallet-widget/) for more details about the widget.
+
+### LogicSignature Transaction
+
+To create a lsig transaction, you can follow steps from the from the offical documentation: [Create Logic Signature Transaction](https://developer.algorand.org/docs/get-details/dapps/smart-contracts/frontend/smartsigs/).
+Here is a glimpse of creating the lsig transaction:
+
+```js
+let fs = require("fs");
+let data = fs.readFileSync(filePath);
+let results = await algodclient.compile(data).do();
+let program = new Uint8Array(Buffer.from(results.result, "base64"));
+let lsig = new algosdk.LogicSigAccount(program);
+
+let lsigTxn = algosdk.makePaymentTxnWithSuggestedParams(
+	lsig.address(),
+	receiver,
+	amount,
+	closeToRemaninder,
+	note,
+	params
+);
+```
+
+After creating the transaction, you can sign it using different wallets.
+
+1. AlgoSigner<br/>
+   We use `connectAlgoSigner()` method to connect the AlgoSigner wallet. Next, we create an instance of the `WebMode` class by passing `AlgoSigner` and the Chain Name i.e the network name. The `web` instance will allow us to sign and send transactions to the blockchain.
+
+```js
+const web = new WebMode(AlgoSigner, CHAIN_NAME);
+```
+
+Webmode method `signLogicSignatureTxn` which is used to sign the txn, expects transaction and LogicSigAccount as parameters.
+To sign the lsig txn:
+
+```js
+const signedTxnResponse = await web.signLogicSignatureTxn(lsigTxn, lsig);
+```
+
+After signing, the returned response is the object containing blob key, which we can further use to send the signed transaction to the network.
+To send the above txn we need to first convert it to base64:
+
+```js
+const base64Txn = Buffer.from(signedTxnResponse.blob).toString("base64");
+const sendTxnResponse = await webmode.sendAndWait(base64Txn);
+```
+
+The sent txn response includes the confirmed transaction response along with txID.
+
+2. MyAlgo Wallet <br/>
+   We use `connectMyAlgoWallet()` method to connect to My Algo wallet.
+
+```js
+   async connectMyAlgoWallet() {
+      try {
+        let walletURL = {
+          token: "",
+          server: "https://node.algoexplorerapi.io",
+          port: "",
+        }
+        let myAlgoMode = new MyAlgoWalletSession(walletURL);
+        await myAlgoMode.connectToMyAlgo();
+        if (myAlgoMode.accounts.length) {
+          this.walletAddress = myAlgoMode.accounts[0].address;
+        }
+      } catch (e) {
+        console.error(e);
+      }
+   }
```

Comment:
why we don't have this function in the `WebMode`?

Reply:
Do you mean catching the error and throwing it?

Reply:
nevermind, probably it's no worth